gems/canvas_sanitize/lib/canvas_sanitize/canvas_sanitize.rb

```diff
@@ -90,6 +90,7 @@ def self.clean_style_value(config, value)
                    'role',
                    'lang',
                    'dir',
+                   :data,
```

Comment:
@dgynn It'd be nice if this maintained consistency with the surrounding entries in the array, i. e. a string rather than a symbol.